### ISSUE TITLE
Readme quickstart Install fails

### ISSUE THREAD

**Issue by kaustavha** [textileio/js-powergate-client#28]
**Describe the bug**

Trying the quickstart in README.md, this cmd fails:   
` npm i --save @textile/powergate-client`  
with error: 
```
> @textile/powergate-client@0.1.0-beta.13 postinstall /Users/kaustavhaldar/Dev/hackfilecoin/node_modules/@textile/powergate-client
> npm run powergate-docker


> @textile/powergate-client@0.1.0-beta.13 powergate-docker /Users/kaustavhaldar/Dev/hackfilecoin/node_modules/@textile/powergate-client
> ./powergate-docker.sh

Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     33      0 --:--:-- --:--:-- --:--:--    33
curl: (6) Could not resolve host: is
curl: (6) Could not resolve host: no
curl: (6) Could not resolve host: longer
curl: (6) Could not resolve host: a
curl: (6) Could not resolve host: supported
curl: (6) Could not resolve host: option.
curl: (6) Could not resolve host: is
curl: (6) Could not resolve host: no
curl: (6) Could not resolve host: longer
curl: (6) Could not resolve host: a
curl: (6) Could not resolve host: supported
curl: (6) Could not resolve host: option..zip
curl: (6) Could not resolve host: is
curl: (6) Could not resolve host: no
curl: (6) Could not resolve host: longer
curl: (6) Could not resolve host: a
curl: (6) Could not resolve host: supported
curl: (6) Could not resolve host: option..zip
Archive:  powergate-docker-v-Xbootclasspath
  End-of-central-directory signature not found.  Either this file is not
  a zipfile, or it constitutes one disk of a multi-part archive.  In the
  latter case the central directory and zipfile comment will be found on
  the last disk(s) of this archive.
unzip:  cannot find zipfile directory in one of powergate-docker-v-Xbootclasspath or
        powergate-docker-v-Xbootclasspath.zip, and cannot find powergate-docker-v-Xbootclasspath.ZIP, period.
```

Traced it to this line in `./powergate-docker.sh`:
```
$ npx json -f package-lock.json dependencies.@textile/grpc-powergate-client.version
-Xbootclasspath is no longer a supported option.
Error: Could not create the Java Virtual Machine.
Error: A fatal exception has occurred. Program will exit.
```

**Comment by asutula** [comment]
Thanks for the report. I'll get this fixed in an update I'm pushing out today.

**Comment by asutula** [comment]
The larger update I'm working on is taking longer than expected, so just fixed this problem directly and published `@textile/powergate-client@0.1.0-beta.14` which includes the fix. Give it a try @kaustavha

**Comment by kaustavha** [comment]
Thanks @asutula  0.1.0 seems to mostly work!

Although trying to upload an image seems to consistently return a fail but im not sure how much of that is just the underlying localnet and pow instance. 
Src - https://github.com/dattasiddhartha/crypto-steganography/pull/2/files

I do get a `(node:14961) UnhandledPromiseRejectionWarning: error code 12 - unknown method Stage for service ffs.rpc.RPCService` with 0.2.0 trying to run `ffs.stage` from the quickstart though.
